### PR TITLE
Add memory result for Artik053

### PR DESCRIPTION
**Pull request by hs0225**
JSRemoteTest-DCO-1.0-Signed-off-by: Hosung Kim hs852.kim@samsung.com

### CODE REVIEW

File: API/testrunner/testrunner.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ def __run_test_on_device(self, app, device, testset, test):
         '''
         testfile = utils.join(device.get_test_path(), testset, test['name'])
 
-        if device.get_type() is 'stm32f4dis':
+        if device.get_type() is 'stm32f4dis' or device.get_type() is 'artik053':
```
Comment (rtakacs) [inline API/testrunner/testrunner.py]: A shorter solution: `if device.get_type() in ['stm32f4dis', 'artik053']`

**Comment by rtakacs** [comment]
Nice, LGTM!

File: API/testrunner/testrunner.py (hunk 1 above)
Comment (hs0225) [reply API/testrunner/testrunner.py]: Thanks for the review. I update.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**